routes.go

```diff
@@ -0,0 +1,184 @@
+	"github.com/xmidt-org/themis/xmetrics/xmetricshttp"
+	"go.uber.org/fx"
+)
+
+var (
+	ServerLabel = "Mimisbrunnr"
+)
+
+type ServerChainIn struct {
+	fx.In
+
+	RequestCount     *prometheus.CounterVec   `name:"server_request_count"`
+	RequestDuration  *prometheus.HistogramVec `name:"server_request_duration_ms"`
+	RequestsInFlight *prometheus.GaugeVec     `name:"server_requests_in_flight"`
+
+	AuthChain *alice.Chain `name:"auth_chain"`
+}
+
+func provideServerChainFactory(in ServerChainIn) xhttpserver.ChainFactory {
+	return xhttpserver.ChainFactoryFunc(func(name string, o xhttpserver.Options) (alice.Chain, error) {
+		var (
+			curryLabel = prometheus.Labels{
+				ServerLabel: name,
+			}
+
+			serverLabellers = xmetricshttp.NewServerLabellers(
+				xmetricshttp.CodeLabeller{},
+				xmetricshttp.MethodLabeller{},
+			)
+		)
+
+		requestCount, err := in.RequestCount.CurryWith(curryLabel)
+		if err != nil {
+			return alice.Chain{}, err
+		}
+
+		requestDuration, err := in.RequestDuration.CurryWith(curryLabel)
+		if err != nil {
+			return alice.Chain{}, err
+		}
+
+		requestsInFlight, err := in.RequestsInFlight.CurryWith(curryLabel)
+		if err != nil {
+			return alice.Chain{}, err
+		}
+
+		if name == "servers.primary" {
+			return in.AuthChain.Append(
+				xmetricshttp.HandlerCounter{
+					Metric:   xmetrics.LabelledCounterVec{CounterVec: requestCount},
+					Labeller: serverLabellers,
+				}.Then,
+				xmetricshttp.HandlerDuration{
+					Metric:   xmetrics.LabelledObserverVec{ObserverVec: requestDuration},
+					Labeller: serverLabellers,
+				}.Then,
+				xmetricshttp.HandlerInFlight{
+					Metric: xmetrics.LabelledGaugeVec{GaugeVec: requestsInFlight},
+				}.Then,
+			), nil
+		}
+		return alice.New(
+			xmetricshttp.HandlerCounter{
+				Metric:   xmetrics.LabelledCounterVec{CounterVec: requestCount},
+				Labeller: serverLabellers,
+			}.Then,
+			xmetricshttp.HandlerDuration{
+				Metric:   xmetrics.LabelledObserverVec{ObserverVec: requestDuration},
+				Labeller: serverLabellers,
+			}.Then,
+			xmetricshttp.HandlerInFlight{
+				Metric: xmetrics.LabelledGaugeVec{GaugeVec: requestsInFlight},
+			}.Then,
+		), nil
+	})
+}
+
+type PrimaryRouter struct {
+	fx.In
+	Router  *mux.Router   `name:"servers.primary"`
+	Handler store.Handler `name:"setHandler"`
+}
+
+type SetRoutesIn struct {
+	fx.In
+	Handler store.Handler `name:"setHandler"`
+}
+type GetRoutesIn struct {
+	fx.In
+	Handler store.Handler `name:"getHandler"`
+}
+type DeleteRoutesIn struct {
+	fx.In
+	Handler store.Handler `name:"deleteHandler"`
+}
+
+type PostRoutesIn struct {
+	fx.In
+	Handler store.Handler `name:"postHandler"`
+}
+
+type GetAllRoutesIn struct {
+	fx.In
+	Handler store.Handler `name:"getAllHandler"`
+}
+
+type PostEventRouteIn struct {
+	fx.In
+	Handler store.Handler `name:"eventHandler"`
+}
+
+func BuildPrimaryRoutes(router PrimaryRouter, sin SetRoutesIn, gin GetRoutesIn, din DeleteRoutesIn, pin PostRoutesIn, gain GetAllRoutesIn, pein PostEventRouteIn) {
```

Comment:
I don't think any of these routes exist.  This looks like argus code; have you updated this file yet? This is where you need to tell the application what function to use for each method+endpoint that mimisbrunnr supports.